### PR TITLE
Projects Endpoints

### PR DESCRIPTION
**Pull request by javipacheco**

### CODE REVIEW

File: server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/XefRoutes.kt [hunk 1]
```diff
@@ -114,3 +114,9 @@ private fun ApplicationCall.getProvider(): Provider =
 
 fun ApplicationCall.getToken(): String =
     principal<UserIdPrincipal>()?.name ?: throw XefExceptions.AuthorizationException("No token found")
+
```
Comment (Montagon) [inline server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/XefRoutes.kt:117]: :heart:

File: server/src/main/kotlin/com/xebia/functional/xef/server/services/RepositoryService.kt [hunk 2]
```diff
@@ -20,3 +24,6 @@ object RepositoryService {
     }
 
 }
+
+fun String.getUser() =
```
Comment (Montagon) [inline server/src/main/kotlin/com/xebia/functional/xef/server/services/RepositoryService.kt:28]: I like the idea, but in this case, I would say that makes more sense to wrap the String value in a Token class or something similar. I don't know if is good to add a function to a primitive type which depends on a BD to work.

**Review by Montagon — CHANGES_REQUESTED** [review]
Should you update the postman collection in order to add those endpoints?

File: server/src/main/kotlin/com/xebia/functional/xef/server/services/RepositoryService.kt (hunk 2 above)
Comment (javipacheco) [reply server/src/main/kotlin/com/xebia/functional/xef/server/services/RepositoryService.kt:28]: Sure!

**Review by Montagon — APPROVED** [review]
🚀

**Pull request merged**